examples/golang/GoLangS3AWS/GoLangs3AWS.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+)
+
+var (
+	s3session *s3.S3
+	//Creating a new S3 session
+)
+
+const (
+	//name of previously created bucket. This bucket was created using the command
+	//MON=1 OSD=1 MDS=0 MGR=0 RGW=1 ../src/vstart.sh -n -d  for starting the cluster then
+	//s3cmd --no-ssl --host=localhost:8000 --host-bucket="localhost:8000/%(bucket)" \
+	//--access_key=0555b35654ad1656d804 \
+	//--secret_key=h7GhxuBLTrlhVUyxSPUKUV8r/2EI4ngqJxD7iBdBYLhwluN30JaT3Q== \
+	//mb s3://bucketname
+	BUCKET_NAME = "cephbucket"
+	REGION      = "us-west-2"
+)
+
+func init() {
+	s3session = s3.New(session.Must(session.NewSession(&aws.Config{
```

Comment:
your code rely on credentials that were configured as part of `aws configure`
please make sure that credentials are provided explicitly in the code

Reply:
ok, trying to get this.